checks.d/disk.py

```diff
@@ -187,7 +187,19 @@ def _collect_part_metrics(self, part, usage):
 
     def _collect_inodes_metrics(self, mountpoint):
         metrics = {}
-        inodes = os.statvfs(mountpoint)
+        # we need to timeout this, too.
+        try:
+            inodes = timeout(5)(os.statvfs)(mountpoint)
+        except TimeoutException:
+            self.log.warn(
+                u"Timeout while retrieving the disk usage of `%s` mountpoint. Skipping...",
+                mountpoint
+            )
+            return metrics
+        except Exception as e:
+            self.log.warn("Unable to get disk metrics for %s: %s", mountpoint, e)
+            return metrics
+
```

Comment:
@yannmh I think you're looking at the wrong thing. These are the `system.fs.inodes.*` metrics. They're not referenced anywhere else.